### ISSUE TITLE
When language set to zh_CN or zh_TW, can't click any item in the left tree panel of 'preferences'.

### ISSUE THREAD

**Issue by sdshq** [doublecmd/doublecmd#1877]
### Version

Double Commander
Version: 1.1.18 gamma
Revision: 568
Commit: c45365d1b
Build date: 2024/09/21
Lazarus: 3.5.0.0
FPC: 3.2.2
Platform: aarch64-Darwin-cocoa
OS version: Mac OS X 15.0.0

### Description

This issue happens since v1.1.17.
When language set to zh_CN or zh_TW, can't click any item in the left tree panel of 'preferences'.
When using other languages, it seems fine.

**Comment by rich2014** [comment]
@sdshq 

can't be reproduced on macOS 12.

can you test it on another computer?

can you provide a screen recording?

**Comment by alabotski** [comment]
```
Double Commander
Version: 1.2.0 alpha
Revision: 11411
Commit: fcca7ac78
Build date: 2024/08/12
Lazarus: 3.99.0.0
FPC: 3.2.2
Platform: x86_64-Darwin-cocoa
OS version: Mac OS X 14.5.0
```

```
Double Commander
Version: 1.2.0 alpha
Revision: 11529
Commit: ea75da6f0
Build date: 2024/10/01
Lazarus: 3.99.0.0
FPC: 3.2.2
Platform: x86_64-Darwin-cocoa
OS version: Mac OS X 14.6.1
```

I have the some problem

**Comment by alabotski** [comment]
When I change Russian to English, all workds fine

**Comment by rich2014** [comment]
@alexx2000 Alex, do you have any idea?

**Comment by alexx2000** [comment]
No, I can't reproduce it too (macOS 11). Maybe problem exists only with newer macOS.

**Comment by rich2014** [comment]
@alexx2000 but why is it related to language?

**Comment by rich2014** [comment]
@sdshq @alabotski 

could you test with Korea at the last line?

**Comment by alexx2000** [comment]
No idea.

**Comment by alabotski** [comment]
I'm on vacation, don't close the ticket.

**Comment by sdshq** [comment]
> @sdshq @alabotski
> 
> could you test with Korea at the last line?

Korean does not have this problem on my mac.

**Comment by sdshq** [comment]
> @sdshq
> 
> can't be reproduced on macOS 12.
> 
> can you test it on another computer?
> 
> can you provide a screen recording?


https://github.com/user-attachments/assets/d1637d79-d544-421d-9837-e054b78a2646

**Comment by rich2014** [comment]
@sdshq 

it's too weird.

could you try the snapshoot from https://doublecmd.sourceforge.io/site/eng/snapshots.php?

**Comment by sdshq** [comment]
@rich2014 

Still, no luck.

Love the new UI by the way.

**Comment by rich2014** [comment]
@sdshq 

what happens if you overwrite `doublecmd.zh_CN.po` with `doublecmd.pot`?

the new UI indeed makes DC significantly native

**Comment by rich2014** [comment]
works fine on macOS 13 too.

it's really weird.

**Comment by sdshq** [comment]
> @sdshq
> 
> what happens if you overwrite `doublecmd.zh_CN.po` with `doublecmd.pot`?
> 
> the new UI indeed makes DC significantly native

That works... So it looks like the file content matters.

**Comment by rich2014** [comment]
@sdshq 

can you locate the problem through binary elimination (二分法)?

**Comment by rich2014** [comment]
i test on macOS 15, and it works fine too.

**Comment by alabotski** [comment]
Double Commander
Version: 1.2.0 alpha
Revision: 11702
Commit: db7dfa19f
Build date: 2024/11/27
Lazarus: 4.99.0.0
FPC: 3.2.2
Platform: x86_64-Darwin-cocoa
OS version: Mac OS X 14.6.1


Problem still present

**Comment by rich2014** [comment]
@alabotski 

can't be reproduced on macOS 11 / 12 / 14.

can you locate the problem in the content through binary elimination (二分法)?

**Comment by alabotski** [comment]
@rich2014 
The problem is reproducible

```
Double Commander
Version: 1.2.0 alpha
Revision: 11790
Commit: e853cdadd
Build date: 2025/01/16
Lazarus: 4.99.0.0
FPC: 3.2.2
Platform: x86_64-Darwin-cocoa
OS version: Mac OS X 14.6.1
```

<img width="253" alt="Image" src="https://github.com/user-attachments/assets/a71c6381-eaa4-4c82-9a35-6586c7b91620" />
